rules/okta_rules.yaml

```diff
@@ -0,0 +1,170 @@
+#Example Rule on login in to OKTA. Disabled by default since it might be noisy
+#- rule: User logged in to OKTA
+#  desc: Detect the user login in to OKTA
+#  condition: okta.evt.type = "user.session.start" 
+#  output: "A user has logged in toOKTA (user=%okta.actor.name, ip=%okta.client.ip)"
+#       priority: NOTICE
+#  source: okta
+#  tags: [okta]
+
+- rule: User Changing password in to OKTA
```

Comment:
I think overall the rules do a good job at showing up what info the plugin provides, but we should decide whether the rules are supposed to be opinionated and look for suspicious/alarming events, or rather provide a changelog of information where a user would do additional analysis on the changelog to find suspicious/alarming events.

The distinction is important because I've seen users, for example, just blindly enable all the k8s audit rules "to provide the best coverage", and then they're drowning in falco alerts that denote all changes to their k8s cluster. And then they blame falco for all the "noise".

I think we should make this distinction at least using rule tags and comments.

Reply:
thanks for your review.
All the rules due to the types of logs and the type of Okta service (IAM) are basically audit rules. 
The response for each rule might change user by user based on what they think it's important for their environment. Hard to fine a one fit all behaviour.
For instance here is the suggestion from OKTA on what it should be monitored from the security prospective: https://github.com/OktaSecurityLabs/CheatSheets/blob/master/SecurityEvents.md
Some of the one proposes are even more noisy and I haven't included that already (login and log out logs).
I can add an "audit" tag on some of them hard to find which one isn't audit due to the log source.

Reply:
@mstemm I disabled by default some rules leaving the most important on security side active. 
The all noisy ones should be disabled by default to reduce the noise as you suggested. 
Thanks for the feedback and let me know your thoughts